README.md

```diff
@@ -74,6 +75,7 @@ Upper-level frameworks or libraries that are powered by Rspack or connected to R
 - [rsbuild-react-chakra-starter](https://github.com/sozonome/rsbuild-react-chakra-starter): initialize react app with rsbuild, Chakra UI and TypeScript setup. Configured with awesome toolings: Biome, Husky + Lint-Staged, Commitlint, and Turbo.
 - [template-rsbuild-react-ts-tailwind](https://github.com/RoyRao2333/template-rsbuild-react-ts-tailwind): Rsbuild starter template with React + Typescript + TailwindCSS + Biome.
 - [rsbuild-turborepo-react-module-federation](https://github.com/nguyenbatranvan/rspack-turbo-module-federation): Rsbuild + Turborepo + Biome + Zustand + React for module federation.
+- [@electron-rsbuild/core](https://github.com/electron-rsbuild/electron-rsbuild): Explore the next generation electron build tooling based on Rsbuild.
```

Comment:
electron-rsbuild has been added to `Ecosystem`, so there is no need to add it to other sections.

```suggestion
```